### PR TITLE
Feature: Implement callable Cloud Functions support (attempt 3 of #219)

### PR DESCRIPTION
**Pull request by rostopira**

### CODE REVIEW

**Comment by gazialankus** [comment]
Just wanted to say thank you! I'm using your branch in my app so I can call Firebase functions in Flutter Web.

File: test/functions_test.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Functions Test</title>
+</head>
+<body>
+<script src="https://www.gstatic.com/firebasejs/6.4.0/firebase-app.js"></script>
```
Comment (kevmoo) [inline test/functions_test.html]: Bump this to 6.6.1 ?

File: test/functions_test.dart [hunk 2]
```diff
@@ -0,0 +1,39 @@
+@TestOn('browser')
```
Comment (kevmoo) [inline test/functions_test.dart]: What needs to be setup for this test to work?

**Comment by kevmoo** [comment]
Ping?

**Pull request closed without merging**